src/pytrajplot/plotting/plot_map.py

```diff
@@ -1,12 +1,19 @@
 """Generate Map Plot Figure."""
 
 # Standard library
+from itertools import groupby
 from pathlib import Path
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+from typing import Union
```

Comment:
Since python 3.10, `Union[A, B]` can also be written as `A | B`

Reply:
Thank you for the input. I wanted to change it but unfortunately, my IDE displays this as an error and does not even allow me to commit the changes besides Python 3.11 is installed in the env and no runtime error is occuring.
Any ideas how to fix this?
 
![dict](https://github.com/MeteoSwiss-APN/pytrajplot/assets/41451360/0b4e19c5-bc6c-4df8-81bc-c625632105d5)
![union_error](https://github.com/MeteoSwiss-APN/pytrajplot/assets/41451360/32578a58-30a1-42c8-bd9d-7e69655c7554)